src/OpenTelemetry/Resources/Resource.cs

```diff
@@ -51,7 +51,7 @@ public Resource(IEnumerable<KeyValuePair<string, string>> labels)
         /// <summary>
         /// Gets the collection of key-value pairs describing the resource.
         /// </summary>
-        public IEnumerable<KeyValuePair<string, string>> Labels { get; }
+        public IEnumerable<KeyValuePair<string, string>> Attributes { get; }
```

Comment:
Attributes per new spec could be string, long, double and bool. We use object everywhere. It also requires validation changes

Reply:
Would you rather do that in the same rename PR or have a follow up PR for that stuff?
Also, is this something I can give a try or is it something you've already covered? If the latter I can pick some other work

Reply:
I agree with @bruno-garcia we can make renaming first. But it's fair that the next step will be to understand whether we need objects as values or specific types.

Reply:
Sure, but then don't accidentally close #375. I believe it mentions extra values types support

Reply:
Changed the description not to auto close.